### PR TITLE
Scene setup and top bars

### PR DESCRIPTION
**Pull request by aaronabraham311**
- Added environment bar
- Modified navbar to have a home icon based on prop passed
- Design looks ugly, but functionality is all we care about right now
- Added routes for environment

Kapture 2021-04-17 at 14 50 53

### CODE REVIEW

File: src/Components/Admin/EnvironmentEditor/environment.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+export default function EnvironmentEditor() {
+    const classes = useStyles();
+
+    return (
+        <div>
+            <Navbar home />
+            <div className={classes.page}>
+                <EnvironmentBar />
+                Hello
```
Comment (ahmedhamodi) [inline src/Components/Admin/EnvironmentEditor/environment.js:21]: Can remove this line, isn't even showing in the loom video (probably needs some padding top).

Alternatively, can do a <div> below environment bar with a paddingTop that will handle the navbar + env bar heights.

File: src/Components/Admin/EnvironmentEditor/environmentBar.js [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+                        >
+                            {" "}
+                            New Scene{" "}
+                        </Button>
+                    </div>
+                    <div className={classes.buttonWrapperRight}>
+                        <Button className={classes.button}> Preview </Button>
+                        <Button className={classes.button}>
+                            {" "}
+                            Share and Publish{" "}
```
Comment (ahmedhamodi) [inline src/Components/Admin/EnvironmentEditor/environmentBar.js:71]: Let's not use spaces, I would prefer padding-left and padding-right attributes on classes.button

File: src/Components/Admin/EnvironmentEditor/environment.js (hunk 1 above)
Comment (aaronabraham311) [reply src/Components/Admin/EnvironmentEditor/environment.js]: Ah forgot about this, deleted.

File: src/Components/Admin/EnvironmentEditor/environmentBar.js (hunk 2 above)
Comment (aaronabraham311) [reply src/Components/Admin/EnvironmentEditor/environmentBar.js]: This was added automatically by Prettier. Deleted

Comment (ahmedhamodi) [reply src/Components/Admin/EnvironmentEditor/environmentBar.js]: Oh interesting, not sure why Prettier does that. There might be a reason - will explore later.

**Review by ahmedhamodi — APPROVED** [review]
Looks good.

I fixed an issue with routing that is introduced by this PR here: https://github.com/uwblueprint/cdc-frontend/pull/32

Please review, as that PR will need to be merged shortly after this PR.

**Pull request merged**